rke/k8s_rke_system_images.go

```diff
@@ -5563,6 +5563,50 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/kubelet-pause:v0.1.6",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.15.13",
 		},
+		// Enabled in 2.5.12
+		"v1.19.16-rancher1-3": {
```

Comment:
Still new to this, but you might want to put a comment for whether Kubernetes hyperkube was updated (it should have been I think) and what other image versions changed. I just know that would make surfing through this file a lot easier for other devs